parcels/kernel/basekernel.py

```diff
@@ -334,13 +334,13 @@ def load_fieldset_jit(self, pset):
                                         g.chunk_loaded_touched, g.load_chunk)
                 if len(g.load_chunk) > g.chunk_not_loaded:  # not the case if a field in not called in the kernel
                     if not g.load_chunk.flags.c_contiguous:
```

Comment:
Above, we change this to
```suggestion
                    if not g.load_chunk.flags['C_CONTIGUOUS']:
```
but here we leave it as was. Should we also change here (and in the lines below)?